File: diretorio3/vetor4.java
//atividade 30/08/24
//aluno: João Pedro Araújo Inácio

import java.util.Scanner;

public class vetor4 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int[] numbers = new int[6];
        int sumEven = 0;
        int countOdd = 0;

        System.out.println("informe 5 numeros inteiros:");

        for (int i = 0; i < 5; i++) {
            numbers[i] = scanner.nextInt();
        }

        System.out.print("numeros pares digitados: ");
        for (int num : numbers) {
            if (num % 2 == 0) {
                System.out.print(num + " ");
                sumEven += num;
            }
        }
        System.out.println();

        System.out.println("Soma numeros pares: " + sumEven);

        System.out.print("Numeros i­mpares digitados: ");
        for (int num : numbers) {
            if (num % 2 != 0) {
                System.out.print(num + " ");
                countOdd++;
            }
        }
        System.out.println();

        System.out.println("Quantidade numeros i­mpares: " + countOdd);
    }
}

File: diretorio3/vetor4.go
//atividade 30/08/24
//aluno: João Pedro Araújo Inácio

package main

import (
	"fmt"
)

func main() {

	var numbers [5]int
	var sumEven int
	var countOdd int

	fmt.Println("informe 5 numeros inteiros:")

	for i := 0; i < 5; i++ {
		fmt.Scan(&numbers[i])
	}

	fmt.Print("numeros pares digitados: ")

	for _, num := range numbers {
		if num%2 == 0 {
			fmt.Print(num, " ")
			sumEven += num
		}
	}
	fmt.Println()

	fmt.Println("soma numeros pares:", sumEven)

	fmt.Print("numeros impares digitados: ")
	for _, num := range numbers {
		if num%2 != 0 {
			fmt.Print(num, " ")
			countOdd++
		}
	}
	fmt.Println()

	fmt.Println("quantidade numeros impares:", countOdd)
}
